server-proxy.js

```diff
@@ -1,8 +1,15 @@
 const utils = require('./utils');
 const grpc = require('grpc');
-const compose = require('koa-compose');
 
-const interceptors = [];
+const _interceptors = [];
```

Comment:
This shouldn't be global but I've got a fix I haven't pushed yet so I guess we can merge this 1st then I'll push my fix.